starport/pkg/cosmosgen/templates/vuex/store/index.ts.tpl

```diff
@@ -1,4 +1,4 @@
-import { txClient, queryClient } from './module'
+import { txClient, queryClient, MissingWalletError } from './module'
```

Comment:
in below, instead of `.fromPartial()` we can use [baseX](https://github.com/tendermint/starport/blob/88781eea534650eb25282e85f65f7da7cefed0ea/starport/templates/app/stargate/vue/src/store/generated/cosmos/cosmos-sdk/cosmos.bank.v1beta1/module/types/cosmos/bank/v1beta1/bank.js#L5):
https://github.com/tendermint/starport/blob/88781eea534650eb25282e85f65f7da7cefed0ea/starport/pkg/cosmosgen/templates/vuex/store/index.ts.tpl#L36

Reply:
Actually we can't, for some reason baseX is not always populated properly (I think it depends on whether some fields are optional or not) and we need the full possible structure.